src/main/java/uk/gov/justice/digital/delius/service/ConvictionService.java

```diff
@@ -392,11 +392,10 @@ private ProbationStatus probationStatusOf(Offender offender) {
     }
 
     private LocalDate previouslyKnownTerminationDateOf(Offender offender) {
-        if (!probationStatusOf(offender).equals(ProbationStatus.PREVIOUSLY_KNOWN)) {
+        if (probationStatusOf(offender) != ProbationStatus.PREVIOUSLY_KNOWN) {
```

Comment:
this is an enum so no point doing `equals`

Reply:
Sure person understood this but there may be other reasons for choosing that method

Reply:
Interesting.  I don't normally work in this code base, so are you suggesting that doing a `!` and `equals` for enums is preferable here?